### ISSUE TITLE
No focus style

### ISSUE THREAD

**Issue by lc512k** [Financial-Times/next-video-page#70]
This element doesn't seem to have any discernible focus styles (tabbing through the page, focus is "lost" for one tab)

![screen shot 2017-02-16 at 11 59 45](https://cloud.githubusercontent.com/assets/3425322/23020566/8c16c9c2-f43f-11e6-8719-3603442007a8.png)

**Comment by lc512k** [comment]
Focus now skips the video altogether, which solves the lost tab issue, but users will want to focus on that play button or at least on the video and not be able to (even though The New Downstairs links to the same place, it's not evident)

Could we make the play button be focusable?
or,
Give the whole video box a focus style?

@i-like-robots

**Comment by ironsidevsquincy** [comment]
Yes, currently it replicates n-teaser, i.e. no focus on the image/play icon, which makes sense as it's just clutter (same link as the title)

Best solution would be to have a single link around the image and title. We might be able to do this for the hero, not sure how feasible that is for the teasers

**Comment by lc512k** [comment]
> which makes sense as it's just clutter (same link as the title)

In the regular teaser, yes. As a video teaser no, since the thing that looks like a play button isn't really a button the user can focus on, so that'll be confusing.

> We might be able to do this for the hero, not sure how feasible that is for the teasers

It'd be really helpful if the whole box could receive focus, both for the hero and the teasers

Is this something that you could pick up, @ironsidevsquincy, or anyone in the video team or working with n-teaser?
